package.json

```diff
@@ -8,8 +8,11 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@popperjs/core": "^2.9.2",
+    "bootstrap": "^5.0.0",
```

Comment:
Since we are downloading bootstrap from the CDN in public/index.html, we don't need to add it as a package here. Let's make a ticket now for converting from CDN usage to npm package usage later on.